hbase-server/src/main/java/org/apache/hadoop/hbase/HDFSBlocksDistribution.java

```diff
@@ -228,33 +228,49 @@ public long getUniqueBlocksTotalWeight() {
    * Implementations 'visit' hostAndWeight.
```

Comment:
nit: `HostAndWeight` has `host` variable which is just set by constructor. We can make it final with this PR.

Reply:
Goop point.